images/cli-artifacts/Dockerfile.rhel

```diff
@@ -1,11 +1,11 @@
 # This Dockerfile builds an image containing the Mac and Windows version of oc
 # layered on top of the Linux cli image.
-FROM registry.ci.openshift.org/ocp/builder:rhel-7-golang-1.16-openshift-4.8 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9 AS builder
```

Comment:
I don't think we want to switch to rhel8, this should be rhel-7 similarly to https://github.com/openshift/oc/pull/855